docs/examples/extracting_entities.ipynb

```diff
@@ -203,19 +265,38 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 25,
+   "execution_count": 5,
    "metadata": {},
    "outputs": [
     {
      "name": "stderr",
      "output_type": "stream",
      "text": [
-      "/home/zayd/workspace/guardrails/guardrails/validatorsattr.py:285: UserWarning: Validator 1-indexed is not valid for element integer.\n",
-      "  warnings.warn(\n",
+      "/Users/zaydsimjee/workspace/guardrails/docs/examples/.venv/lib/python3.10/site-packages/guardrails/validatorsattr.py:307: UserWarning: Validator 1-indexed is not installed!\n",
+      "  warnings.warn(f\"Validator {validator_name} is not installed!\")\n",
+      "/Users/zaydsimjee/workspace/guardrails/docs/examples/.venv/lib/python3.10/site-packages/guardrails/validator_base.py:410: FutureWarning: Accessing `LowerCase` using\n",
+      "`from guardrails.validators import LowerCase` is deprecated and\n",
+      "support will be removed after version 0.5.x. Please switch to the Guardrails Hub syntax:\n",
+      "`from guardrails.hub import LowerCase` for future updates and support.\n",
+      "For additional details, please visit: https://hub.guardrailsai.com/validator/guardrails/lowercase.\n",
+      "\n",
+      "  warn(\n",
+      "/Users/zaydsimjee/workspace/guardrails/docs/examples/.venv/lib/python3.10/site-packages/guardrails/validator_base.py:410: FutureWarning: Accessing `TwoWords` using\n",
+      "`from guardrails.validators import TwoWords` is deprecated and\n",
+      "support will be removed after version 0.5.x. Please switch to the Guardrails Hub syntax:\n",
+      "`from guardrails.hub import TwoWords` for future updates and support.\n",
+      "For additional details, please visit: https://hub.guardrailsai.com/validator/guardrails/two_words.\n",
       "\n",
-      "/home/zayd/workspace/guardrails/guardrails/validatorsattr.py:285: UserWarning: Validator percentage is not valid for element float.\n",
-      "  warnings.warn(\n",
-      "\n"
+      "  warn(\n",
+      "/Users/zaydsimjee/workspace/guardrails/docs/examples/.venv/lib/python3.10/site-packages/guardrails/validator_base.py:410: FutureWarning: Accessing `OneLine` using\n",
+      "`from guardrails.validators import OneLine` is deprecated and\n",
+      "support will be removed after version 0.5.x. Please switch to the Guardrails Hub syntax:\n",
+      "`from guardrails.hub import OneLine` for future updates and support.\n",
+      "For additional details, please visit: https://hub.guardrailsai.com/validator/guardrails/one_line.\n",
+      "\n",
+      "  warn(\n",
+      "/Users/zaydsimjee/workspace/guardrails/docs/examples/.venv/lib/python3.10/site-packages/guardrails/validatorsattr.py:307: UserWarning: Validator percentage is not installed!\n",
+      "  warnings.warn(f\"Validator {validator_name} is not installed!\")\n"
```

Comment:
All of these are likely because the `format` properties in the rail spec need to be updated to use the hub